server/www/decoybutton.html

```diff
@@ -0,0 +1,102 @@
+<a href="https://geekprank.com/fake-virus/"><button type="button" class="btn btn-outline-danger">YOUR COMPUTER HAS A VIRUS CLICK HERE TO REMOVE</button></a>
+<button type="button" class="btn btn-outline-warning">WARNING CLICK HERE</button>
+<button type="button" class="btn btn-outline-info">Info</button>
+<button type="button" class="btn btn-outline-light">Accept</button>
+<button type="button" class="btn btn-outline-dark">Begin Download</button> <br> <br>
+<div class="btn-group btn-group-toggle" data-toggle="buttons">
+  <label class="btn btn-secondary active">
+    <input type="radio" name="options" id="option1" autocomplete="off" checked> Purchase Ticket
+  </label>
+  <label class="btn btn-secondary">
+    <input type="radio" name="options" id="option2" autocomplete="off"> Change Date of Flight
+  </label>
+  <label class="btn btn-secondary">
+    <input type="radio" name="options" id="option3" autocomplete="off"> Seat Request
+  </label>
+</div>
+<br><br>
+<button type="button" class="btn btn-primary">Download</button>
+<button type="button" class="btn btn-secondary">Download now</button>
+<button type="button" class="btn btn-success">Download</button>
+<button type="button" class="btn btn-danger">Click here to download</button>
+<a href = "https://uxdesign.cc/dark-patterns-in-ux-design-7009a83b233c"><button type="button" class="btn btn-warning">CLICK HERE</button></a>
+<button type="button" class="btn btn-info">Download Now</button>
+<button type="button" class="btn btn-light">Download Ticket</button>
+<button type="button" class="btn btn-dark">Accept Purchase</button>
+<button type="button" class="btn btn-link">Click link</button>
+<a class="btn btn-primary" role="button">Link</a>
+<br><br>
+<button class="btn btn-primary" type="submit">Button</button>
+<input class="btn btn-primary" type="button" value="Download PDF">
+<input class="btn btn-primary" type="submit" value="Email Ticket">
+<input class="btn btn-primary" type="reset" value="Download">
+<br><br>
+<button type="button" class="btn btn-primary">Download</button>
+<button type="button" class="btn btn-secondary">BOOK FLIGHT</button>
+<button type="button" class="btn btn-success">Download Ticket</button>
+<button type="button" class="btn btn-danger">Click here to download</button>
+<button type="button" class="btn btn-warning">CLICK HERE</button>
+<button type="button" class="btn btn-info">CONFIRM TICKET</button>
+<button type="button" class="btn btn-light">CLICK HERE NOW</button>
+<a href = "https://www.darkpatterns.org/types-of-dark-pattern"> <button type="button" class="btn btn-dark">Accept Purchase</button></a>
+ <br><br><br>
+<button type="button" class="btn btn-link">CLICK TO DOWNLOAD</button>
+<br><br>
+<a href="https://www.nytimes.com/2012/02/19/magazine/shopping-habits.html?pagewanted=all"><button type="button" class="btn btn-primary btn-lg btn-block">CLICK HERE FOR A FREE TICKET</button></a>
```

Comment:
just a thought (but feel free to disagree), but what if these links were set to open in new tabs (by adding the `target="_blank"` attribute to the `a` tags). this way u don't "leave" the game, but rather take more of an "aside"

Reply:
I think that's a good idea, I'll switch it in my code.